### ISSUE TITLE
Run validation at train step = 0

### ISSUE THREAD

**Issue by gideonite** [ai2cm/ace#216]
Training appears to make a lot of progress at the beginning of training and slow down quite significantly. We originally thought that this may be due to a rapidly decaying learning rate, but some of the results of the fine-tuning experiments suggest otherwise. It's hard to tell how much progress we are making early on in training without an initial value for our metrics.
